recipes/esmp/build.sh

```diff
@@ -0,0 +1,105 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC ${CFLAGS}"
+export CXXLAGS="${CFLAGS} ${CXXLAGS}"
+export CPPFLAGS="-I${PREFIX}/include ${CPPFLAGS}"
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+export ESMF_DIR=`pwd`"/esmf"
+export ESMP_source="ESMP"
+export ESMF_PTHREADS="OFF"
+export ESMF_OS=`uname -s`
+
+export ESMF_COMPILER="gfortran"
+export ESMF_ABI="64"
+if [ `uname` == Darwin ]; then
+    export ESMF_OPENMP="OFF"
+    export CC="/usr/bin/gcc"
+    export FC=${PREFIX}/bin/gfortran
+    export F77=${PREFIX}/bin/gfortran
+    export F90=${PREFIX}/bin/gfortran
+else
+    export ESMF_OPENMP="ON"
+fi
+
+# OPENMPI bits
+CONDA_LST=`conda list`
+if [[ ${CONDA_LST}'y' == *'openmpi'* ]]; then
```

Comment:
it does smell a lot like it, but i'm not convinced it is. Please advice, the first step would be to have openmpi edited to say it's offering the `openmpi` feature correct? That would be easy enough, but then I'm not 100% positive on how to tell let's say hdf5 to build differently if the "openmpi" feature is present. And how do we tell `conda build` to build both versions of hdf5? thanks for any pointer on this, I've been asking to every continuum employee I meet at various conference (including the ones showing off conda) and it seems like features is a bit of an orphan there.